central/complianceoperator/v2/report/manager/complianceReportgenerator/consts.go

```diff
@@ -3,10 +3,10 @@ package complianceReportgenerator
 const (
 	defaultEmailBodyTemplate = "{{.BrandedPrefix}} has scanned your clusters for compliance with the profiles in your scan configuration." +
 		"The attached report lists the checks performed and provides corresponding details to help with remediation. \n" +
-		"Profiles:{{.Profile}} \n" +
-		"Passing:{{.Pass}} checks \n" +
-		"Failing:{{.Fail}} checks \n" +
-		"Mixed:{{.Mixed}} checks \n" +
+		"Profiles:{{.Profile}}| \n" +
+		"Passing:{{.Pass}} checks|\n" +
+		"Failing:{{.Fail}} checks| \n" +
+		"Mixed:{{.Mixed}} checks|\n" +
```

Comment:
Because the newline escape character is not always honored by the email agent, I would add leading as well as trailing spaces around the new pipe `|` characters.

```suggestion
		"Profiles:{{.Profile}} | \n" +
		"Passing:{{.Pass}} checks |\n" +
		"Failing:{{.Fail}} checks | \n" +
		"Mixed:{{.Mixed}} checks |\n" +
```

Reply:
+1